web/src/engine/websites/MonochromeScans.ts

```diff
@@ -0,0 +1,77 @@
+import { Tags } from '../Tags';
+import icon from './MonochromeScans.webp';
+import { Chapter, DecoratableMangaScraper, Manga, type MangaPlugin, Page } from './../providers/MangaPlugin';
+import * as Common from './decorators/Common';
+import { FetchJSON, FetchRequest } from './../FetchProvider';
+
+type APIManga = { title: string, id: string };
+type APIChapter = { name: string, volume: number, number: number, id: string, mangaId: string, version : number, length : number };
+const apiUrl = 'https://api.manga.d34d.one';
+
+@Common.ImageDirect()
+
+export default class extends DecoratableMangaScraper {
+
+    public constructor() {
+        super('monochromescans', `Monochrome Scans`, 'https://manga.d34d.one', Tags.Language.English, Tags.Media.Manga, Tags.Source.Scanlator);
+    }
+
+    public override get Icon() {
+        return icon;
+    }
+
+    public override ValidateMangaURL(url: string): boolean {
+        return /^https?:\/\/manga\.d34d\.one\/manga\//.test(url);
+    }
+
+    public override async FetchManga(provider: MangaPlugin, url: string): Promise<Manga>{
+        const id = url.split('/').pop();
+        const request = new FetchRequest(new URL('/manga/'+ id, apiUrl).href);
+        const data = await FetchJSON<APIManga>(request);
+        return new Manga(this, provider, data.id, data.title.trim());
+    }
+
+    public override async FetchMangas(provider: MangaPlugin): Promise<Manga[]> {
+        const mangaList = [];
+        for (let page = 1, run = true; run; page++) {
+            const mangas = await this.getMangasFromPage(page, provider);
+            mangas.length > 0 ? mangaList.push(...mangas) : run = false;
+        }
+        return mangaList;
+
+    }
+
+    private async getMangasFromPage(page: number, provider: MangaPlugin) {
+        const uri = new URL('/manga', apiUrl);
+        uri.searchParams.set('limit', '50');
```

Comment:
Is `50` the maximum or can it be increased further?

Reply:
It seems so, by default its 10, and more than 50 triggers errors 422.

Reply:
![image](https://user-images.githubusercontent.com/10671518/224480096-d4732ad4-9981-4682-aa8b-2badc04a55d9.png)